COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=EVM server daemon
-After=syslog.target memcached.service
-Wants=memcached.service
+After=memcached.service vmdb-up.service
+Wants=memcached.service vmdb-up.service
```

Comment:
sorry to be the person who says this but...

before running this, we need to pick a better name

Reply:
haha, naming things... classic....

`manageiq-ready.service`

Reply:
This is more specific than a general "manageiq-ready", specifically is checking for the availability of the vmdb database
As we add other external service dependencies we'll need more of these (e.g. kafka / memcached)

`vmdb-ready` would work if you guys don't like `-up.service`

Reply:
or `manageiq-up.service` or even `manageiq-db-up`... calling it `db-up` feels too generic  and `vmdb-up` is clinging to old names that we should gradually move away from.

Reply:
`manageiq-up.service` has the same issue as `manageiq-ready.service`, but `manageiq-db-up.service` would work.  Then we could have `manageiq-kafka-up.service` then

Reply:
Is there a general pattern for the naming suffix (-up vs -ready, I mean)?  I originally was going to comment about how I didn't really like -up, because it sounds more like a verb /command than a question (i.e. "start the vmdb" as opposed to "is the vmdb started?").  -ready sounds better, IMO, but if there's a general pattern for this, I think it's preferable to follow that.

Reply:
No general pattern, we can name it whatever we ant

Reply:
> `manageiq-up.service` has the same issue as `manageiq-ready.service`, but `manageiq-db-up.service` would work. Then we could have `manageiq-kafka-up.service` then

Good point... I like the concept of the various manageiq services living near each other in both service lists but also where it's stored in the filesystem.  It makes it more obvious what services we ship but also what we depend on.